tiny_tds.gemspec

```diff
@@ -22,8 +22,8 @@ Gem::Specification.new do |s|
   s.metadata['msys2_mingw_dependencies'] = 'freetds'
   s.add_development_dependency 'mini_portile2', '~> 2.5.0'
   s.add_development_dependency 'rake', '~> 13.0.0'
-  s.add_development_dependency 'rake-compiler', '~> 1.1.0'
-  s.add_development_dependency 'rake-compiler-dock', '~> 1.1.0'
+  s.add_development_dependency 'rake-compiler', '~> 1.2.0'
+  s.add_development_dependency 'rake-compiler-dock', '~> 1.3.0'
```

Comment:
I wonder if we even need the patch level specifier on these... maybe it'd be best as `'rake-compiler', '~> 1.2'` so the minor version is floating?

Reply:
I removed the minor version constraint for `rake-compiler` in [801f16c](https://github.com/rails-sqlserver/tiny_tds/pull/533/commits/801f16ccf5685eff6817e559858e920e44fd948d). I kept it for `rake-compiler-dock` to keep it aligned with the Docker image, as stated in https://github.com/rails-sqlserver/tiny_tds/pull/533#discussion_r1336300203.